### PR TITLE
added a search field to the multiplayer tab

### PR DESCRIPTION
**Pull request by cdqwertz**
This pull adds a simple search field to the multiplayer tab.

### CODE REVIEW

File: builtin/mainmenu/tab_multiplayer.lua [hunk 1]
```diff
@@ -199,6 +206,19 @@ local function main_button_handler(tabview, fields, name, tabdata)
 		return true
 	end
 
+	if fields.btn_search then
```
Comment (rubenwardy) [inline builtin/mainmenu/tab_multiplayer.lua]:
```lua
or fields.key_enter_field == "search"
```

to add search on enter press

File: builtin/mainmenu/tab_multiplayer.lua [hunk 2]
```diff
@@ -87,6 +87,13 @@ local function get_formspec(tabview, name, tabdata)
 		retval = retval .. ";0]"
 	end
 
+	if tabdata.fav_selected then
+		retval = retval .. "field[0.15,5;7,1;search;;".. (menudata.favorites[tabdata.fav_selected].name or "") .."]"
+	else
+		retval = retval .. "field[0.15,5;7,1;search;;]"
```
Comment (rubenwardy) [inline builtin/mainmenu/tab_multiplayer.lua]: add `;false]` to the end of both of these to disable close on enter press

**Comment by Rogier-5** [comment]
Can you change the commit message to present tense ? Besides being good practise in general, it's also in the [git rules](http://dev.minetest.net/Git_Guidelines).

**Comment by SmallJoker** [comment]
Why did you close it?

**Comment by cdqwertz** [comment]
I think I will make a new pull. Its easier than renaming the commits and squashing them.

**Comment by cdqwertz** [comment]
https://github.com/minetest/minetest/pull/4514

**Comment by Ferk** [comment]
@cdqwertz  if you have only one commit, you can just easily re-commit it, you do not have to add new commits and squash.

    git commit --amend
    git push -f

**Comment by rubenwardy** [comment]
Note: you need to make your changes and git add them first:

```bash
git add .
git commit -m "A fault commit"
git push origin branch
# Oh noes!
# make changes
git add
git commit --amend
git push -f origin branch
```

**Pull request closed without merging**